### ISSUE TITLE
Some problems I run into while using your model

### ISSUE THREAD

**Issue by agil27** [Ruifeng-Zhou24/CompactCNN_for_FER#3]
1. There should be a BatchNorm2d after every convolution, right? or the model won't work. 
2. Why are you using VGG19 instead of your own model in the training?
3. Why are you re-inventing all the transforms defined in torchvision? it's perplexing
4. WIl a Adam optimizer simply work better than your complicated SGD optimizer?
